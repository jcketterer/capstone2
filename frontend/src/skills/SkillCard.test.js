import { render } from '@testing-library/react';
import SkillCard from './SkillsCard';

it('should render without breaking', () => {
  render(<SkillCard skill={{}} />);
});

it('should match snapshot', () => {
  const { asFragment } = render(<SkillCard skill={{}} />);
  expect(asFragment()).toMatchSnapshot();
});
