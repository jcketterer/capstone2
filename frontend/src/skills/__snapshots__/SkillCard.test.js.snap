// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="card"
  >
    <div
      class="card-body"
    >
      <ul
        style="list-style-type: none;"
      >
        <li
          class="lead"
        />
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
